Projects/soupstone.md

```diff
@@ -3,19 +3,95 @@
 * [On Github](https://github.com/jamesmunns/soupstone)
 
 ## Summary
-A fun little electronic fidget toy (eventually) with the help of [Embassy](https://github.com/embassy-rs/embassy)
 
-## Goals
+A fun little electronic fidget toy (eventually). The goal is to make a small, battery powered device that has a bunch of fun inputs and outputs, like clicky switches and a piezo speaker, which responds to your inputs.
+
+Ideally: It should be very easy to hack on, and write fun random-ish behaviors, to make it more interesting to play with.
+
+I am currently using the [Seeed Studio XIAO nRF52840 Sense] as the main MCU/SOM, and [Embassy] for software and driver support.
+
+The name comes from the [Stone Soup] folk story, as well as [worry stones].
 
+[Embassy]: https://github.com/embassy-rs/embassy
+[Seeed Studio XIAO nRF52840 Sense]: https://wiki.seeedstudio.com/XIAO_BLE/
+[Stone Soup]: https://en.wikipedia.org/wiki/Stone_Soup
+[worry stone]: https://en.wikipedia.org/wiki/Worry_stone
 
+## Goals
+
+* Make it quick and easy to iterate on
+* Make it robust enough to carry around and use continuously
+* Don't make it too complicated, it's a toy!
+* Build a few units to share with friends and family, and as a neat gift
+* Maybe make it something nice to use later for teaching embedded Rust
 
 ## Current State
 
+As of March 2023, still pretty early! I've been starting with the [nRF52840-DK], because I have some laying around, and they have both a debugger and the MCU USB device port broken out.
```

Comment:
I think the correct term would be ["lying around" ](https://www.grammarly.com/blog/lay-lie/#:~:text=You%20lie%20down%2C%20but%20you,lieing%E2%80%94beware%20of%20spelling).)

Reply:
"broken out" is a term that generally means "is made available on the specific board".

Often, chips like the one I am using here will have lots of different features, but sometimes for space or price reasons (in case external parts or connectors), not all of the capabilities are fully possible to use.

"broken out" is sorta a shorthand for that. I'll see if I can explain that a bit better in the text.